srcjs/input_binding_slider.js

```diff
@@ -90,13 +122,26 @@ $.extend(sliderInputBinding, textInputBinding, {
         msg.from = data.value;
       }
     }
-    if (data.hasOwnProperty('min'))  msg.min   = data.min;
-    if (data.hasOwnProperty('max'))  msg.max   = data.max;
-    if (data.hasOwnProperty('step')) msg.step  = data.step;
+    var sliderFeatures = ['min', 'max', 'step'];
+    for (var feats of sliderFeatures) {
```

Comment:
Need to check browser support for `for - of`.

Reply:
Support doesn't look good: 
JavaScript MDN: Internet Explorer note\ [supported](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/for...of#Browser_compatibility)
I'll use traditional index iteration, but I could also just go back to how it was.